microdnf/commands/advisory/arguments.hpp

```diff
@@ -64,11 +68,24 @@ class AdvisoryUpdatesOption : public libdnf::cli::session::BoolOption {
               command,
               "updates",
               '\0',
-              _("Show advisories about newer versions of installed packages for which a newer version is available."),
+              _("Show advisories containing newer versions of installed packages for which a newer version is "
+                "available."),
               false) {}
 };
 
 
+class AdvisoryWhatcontainsOption : public libdnf::cli::session::StringListOption {
+public:
+    explicit AdvisoryWhatcontainsOption(libdnf::cli::session::Command & command)
+        : StringListOption(
+              command,
+              "whatcontains",
```

Comment:
Should this option be called something like `--contains-pkg` or `--contains-pkgs`? I don't like the `what` in the name too much and it should at least be `--what-contains`?

Reply:
Changed to `--contains-pkgs` as agreed.